Test on Python 3.7

hugovk opened the pull request:
Fixes #22.

hugovk commented:
Looks like Travis CI isn't yet enabled for this repo.

Here's how it looks on my fork:

* https://travis-ci.org/hugovk/picknmix/builds/559364273

Cheukting commented:
Yup I remove 3.7 as it was not working before for some strange reasons.

Cheukting commented on CONTRIBUTING.rst:
```diff
@@ -102,7 +102,7 @@ Before you submit a pull request, check that it meets these guidelines:
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 2.7, 3.4, 3.5 and 3.6, and for PyPy. Check
+3. The pull request should work for Python 2.7, 3.5+ and for PyPy. Check
```
Mind removing 2.7 for me? This project should only support 3.5+

Cheukting commented:
Thanks! Mind putting your name under contributors in AUTHORS.rst? You can skip the email if you want.

hugovk commented:
Updated!

Yeah, the gotcha is Python 3.7 is only available on with Ubuntu Xenial on Travis CI (see https://github.com/travis-ci/travis-ci/issues/9815, but no longer needs `sudo`).

I also ran the [pyupgrade](https://github.com/asottile/pyupgrade) tool to automatically update some syntax for Python 3+:

```sh
pip install -U pyupgrade
pyupgrade `find . -name "*.py" -type f` --py3-plus
```

And removed the dot in `py.test` as recommended by https://twitter.com/pytestdotorg/status/753767547866972160.

Cheukting commented:
All looks good! Thank you so much. I learnt a lot. :-)

hugovk commented:
You're welcome!
